test/integration/services/patient-quick-search.js

```diff
@@ -1,10 +1,13 @@
 import _ from 'underscore';
-import patientFixture from 'fixtures/collections/patients.json';
+
 import { getRelationship, getIncluded } from 'helpers/json-api';
 
+import fxPatients from 'fixtures/collections/patients';
+
 context('Patient Quick Search', function() {
   beforeEach(function() {
-    const patients = _.map(_.sample(patientFixture, 10), (patient, index) => {
+    const clonedfxPatients = _.extend({}, _.sample(fxPatients, 10));
```

Comment:
this doesn't work `sample(foo, 10)` returns an array. not an object.

could be
```js
_.map(_.sample(patientFixture, 10), (patient, index) => {
  return _.extend({}, patient, {
    id: `${ index }`,
    first_name: 'Test',
....
  });
});
```

Reply:
Good call, that's been updated.